tools/nl/embeddings/build_embeddings.py

```diff
@@ -32,19 +36,33 @@
                     'A dict of user provided embeddings/model catalog')
 
 
+def _init_logger():
```

Comment:
this should be set in admin/html.py instead of here?

Reply:
That does not work because admin/html.py forks build embeddings in the new process.